qiskit/transpiler/passes/group_gates.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Pass for grouping runs of two qubit gates and returning another graph with larger nodes.
+"""
+
+# This will be turned into an analysis pass.  At present this class is only used by a_star_cx.py
```

Comment:
Why not writing it as a pass directly?

Reply:
because the dag needs modifications to support grouped gates.

Reply:
should be moved out the `qiskit/transpiler/passes/` directory?